### DISCUSSION TITLE
ask error

### DISCUSSION THREAD

**Discussion by halobug** [danny-avila/LibreChat#110] [Troubleshooting] [answered]
api_1  | CLIENT RESPONSE {
api_1  |   response: 'Hello there! Did you mean to type BAAI? If so, what can I assist you with regarding that term?',
api_1  |   conversationId: '1f35b103-a084-4246-af3d-****',
api_1  |   parentMessageId: 'd275df1b-bae1-4128-97e7-****',
api_1  |   messageId: 'ae141143-74f6-4df3-b5a5-****',
api_1  |   details: {}
api_1  | }
api_1  | TypeError: Cannot read properties of undefined (reading 'includes')
api_1  |     at ask (/api/server/routes/ask.js:134:25)
api_1  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
api_1  |     at async /api/server/routes/ask.js:51:10
api_1  | The client has disconnected.

**Comment by danny-avila** [comment, marked as answer]
@halobug this looks like a bug from the master branch. are you on main branch? line 134 is looking for a property that is clearly defined in the log, the response.

**Reply by halobug** [reply]
Yes, the main branch. The problem appears to be that the gptResponse.text parameter does not exist

https://github.com/danny-avila/chatgpt-clone/blob/master/api/server/routes/ask.js#L134

**Reply by danny-avila** [reply]
@halobug you're looking at the master branch which is now stale, i see the change live here on main branch and I haven't edited since you started the issue 
https://github.com/danny-avila/chatgpt-clone/blob/main/api/server/routes/ask.js
